ui/components/config/provider-form.tsx

```
@@ -0,0 +1,834 @@
+                    <CardTitle className="flex items-center gap-2 text-base">
+                      <Globe className="h-4 w-4" />
+                      Network Configuration
+                    </CardTitle>
+                  </CardHeader>
+                  <CardContent className="space-y-4 px-0">
+                    <div className="grid grid-cols-1 gap-4">
+                      <div>
+                        <label className="text-sm font-medium">
+                          Base URL{" "}
+                          {baseURLRequired ? "(Required)" : "(Optional)"}
+                        </label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)